ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -328,6 +342,9 @@ class GraphsContainer extends React.Component {
                 tickCount={4}
                 style={axisStyle}
                 tickFormat={this.setLeftPadding}
+                tickLabelComponent={positionedTick}
+                axisLabelComponent={positionedLabel}
+                label={yAxisLabel}
```

Comment:
This might need to change once my pr #5762 lands because of how it utilizes custom tool tips/labels.

Do you have a link for tests that will show this measurement unit? I wasn't able to see these changes just grabbing random tests (running this pr in docker).